pkg/lightning/log/filter.go

```diff
@@ -28,6 +28,12 @@ func NewFilterCore(core zapcore.Core, filteredPackages ...string) *FilterCore {
 	}
 }
 
+// With adds structured context to the Core.
+func (f *FilterCore) With(fields []zapcore.Field) zapcore.Core {
+	f.Core = f.Core.With(fields)
+	return f
```

Comment:
```suggestion
	return &FilterCore{
		Core:    f.Core.With(fields),
		filters: f.filters,
	}
```

`With` is not a mutable method.